docker/kubernetes.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
Can we please keep this script in the worker folder itself. Basically at --
`scripts/workers/code_upload_worker_utils/deploy_calico.sh`